_includes/default-tutorial.html

```diff
@@ -0,0 +1,476 @@
+<h1>Welcome</h1>
```

Comment:
@joelgrus @schmmd FYI I took a couple of tweaks to this intro content:

1. Made sure everything was wrapped in the proper html tags and has the right classes.
2. Reworded the "Before we get started..." part so that it's all in once sentence before showing the command. Breaking it up to before and after the command looked weird to me. Up to you guys how to proceed here, just wanted to call it out.